sdk/common/src/main/java/com/mesosphere/sdk/offer/taskdata/TaskPacking.java

```diff
@@ -0,0 +1,100 @@
+package com.mesosphere.sdk.offer.taskdata;
+
+import org.apache.mesos.Protos.ExecutorID;
+import org.apache.mesos.Protos.ExecutorInfo;
+import org.apache.mesos.Protos.TaskInfo;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+
+/**
+ * Mesos requirements do not allow a TaskInfo to simultaneously have a Command and Executor.  In order to
+ * workaround this we encapsulate a TaskInfo's Command field in an ExecutorInfo and store it in the data field of
+ * the TaskInfo.
+ *
+ * Unpacked:
+ * - taskInfo
+ *   - executor
+ *   - data: custom
+ *   - command
+ *
+ * Packed:
+ * - taskInfo
+ *   - executor
+ *   - data: serialized executorinfo
+ *     - data: custom
+ *     - command
+ */
+public class TaskPacking {
```

Comment:
Maybe have this class name match other similar classes (`TaskPackingUtils` or similar) for guessability.